src/chrome/content/rules/Bytejail.com.xml

```diff
@@ -0,0 +1,9 @@
+<ruleset name="Bytejail.com">
+  <target host="bytejail.com" />
+  <target host="*.bytejail.com" />
+ 
+  <test url="http://bytejail.com/" />
```

Comment:
This test is redundant, as literal targets generate implicit tests. Perhaps you were meaning to test www.bytejail.com?